src/Message/AbstractMessage.php

```diff
@@ -167,18 +167,48 @@ abstract class AbstractMessage extends AbstractPart
         return null !== $references ? \explode(' ', $references) : [];
     }
 
+    /**
+    * Get body Mixed.
+    *
+    * @return string
+    */
+    private function getBodyMixed($iterator): ?string
+    {
+        $mixedHTML = '';
+        foreach ($iterator as $part) {
+            if (self::SUBTYPE_MIXED === $part->getSubtype()) {
+                foreach ($part->getParts() as $mixedPart) {
+                    if(self::SUBTYPE_HTML === $mixedPart->getSubtype()) {
+                        $mixedHTML .= $mixedPart->getDecodedContent();
```

Comment:
Hi, with this you've effectively passed the test, but considering the fix as a whole it seems to me the solution is worsening the issue.

If you don't know the email has multiple HTML parts, with this PR the user gets an invalid HTML syntax (content simplified):
```html
<html><body>First Part HTML Message</body></html>
<html><body>Second part HTML Message</body></html>
<html><body>Last Part HTML Message</body></html>
```

The same issue is in place for `getBodyText`.

The only clear solution I have is to rename `getBody(Html|Text)` to `getFirstBody(Html|Text)`, so the user knows the API returns something that could be partial, and maybe provide a new API to fetch all readable parts.
The current `getBody(Html|Text)` should be marked as deprecated, and to be removed in the next major version.

Reply:
Hi,

I'm aware of the invalid HTML Syntax problem. I assumed that this would be kind of desired because it is the original content of the Mail. I wasn't sure if you wanted the library to manipulate it.

In my own code I'm using `DOMDocument()` to fix this. Maybe another function implementing `DOMDocument()` into the library might be the solution here to.

Kind regards
Kevin

Reply:
Hi @Slamdunk ,

do your have any Updates on this?

Kind regards
Kevin

Reply:
I'd like to retain the current behavior for the already implemented APIs.

If you still want to propose a solution, I'd prefer see new APIs for the issue.

Reply:
Hi @Slamdunk,

I can take a look at it, when I have some spare time. Would the following be in your interest:

1.) New Function to check if there are multiple HTML Parts (returns true/false)
2.) New Function to retrieve those original but invalid HTML Parts (Array)

Regarding the invalid HTML Structure:
Should there also be a function to transform the invalid HTML Structure to a valid Structure?
I'm not sure if there is a generally valid Solution for this. And even if there is, I'm not sure if it should be done by this library.

Reply:
> 1.) New Function to check if there are multiple HTML Parts (returns true/false)

The solution should also consider multiple TEXT/PLAIN parts.

>  I'm not sure if there is a generally valid Solution for this. And even if there is, I'm not sure if it should be done by this library.

Me neither.

Reply:
Regarding the Text/Plain Parts:
I've never seen such a mail. Are you sure this can happen? 

These splitted HTML Parts can only exist if there are embedded attachments; at least as far as I know. I don't see a way how this can happen with a text/plain mail.